src/main/java/com/api/salesreport/salesReport/entities/Lead.java

```diff
@@ -0,0 +1,57 @@
+package com.api.salesreport.salesReport.entities;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.NotNull;
+
+
+@Entity
+public class Lead {
```

Comment:
A relação de um Lead com um Client é através do email somente. A princípio não precisa criar uma relação direta (usando chave estrangeira / client_id). Minhas sugestões de modelagem são:

- Solução simples: Fazer Lead e Client serem independentes (Bad idea! Ignore OOP in Java doesn't make sense :laughing:)
- Solução boa: Criar uma abstração que contenha os campos em comum entre Lead e Client, ambas devem estender essa abstração (talvez `Visitor`?).